frontend/src/metabase/common/components/DataPicker/hooks/useHasModels.ts

```diff
@@ -0,0 +1,6 @@
+import { useSearchQuery } from "metabase/api";
+
+export const useHasModels = (): boolean => {
+  const { data } = useSearchQuery({ models: ["dataset"], limit: 1 });
```

Comment:
@kamilmielnik I think you can run one query (e.g. just with `models: ["card']`) and then check `available_models` for `card`, `dataset`, and `metric` (future)

Reply:
Good idea!
ac7b9679a1e697ad76e6a3f0a6bc1fc122923cba